NWT_SUBMIT/NWTOPT_FILES/objective.py

```diff
@@ -298,18 +299,18 @@ def objective(inputHp):
     # run the model and check for errors
     if not runModel(pathtonwt, initnwt, cwd, timelim, run_command):
         finish_time = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
-        return {'loss': 999999999999,
+        return {'loss': INF,
                 'status':  STATUS_OK,
                 'eval_time': eval_time,
-                'mass_balance': 999999,
+                'mass_balance': INF,
                 'sec_elapsed': timelim,
                 'iterations': -1,
                 'NWT Used': pathtonwt,
                 'finish_time': finish_time}
     # if no errors get run results
     sec_elapsed, iterations, mass_balance = getRunResults(cwd, listfile)
-    if mass_balance == 999999:
-        loss = 999999999999
+    if mass_balance == INF:
```

Comment:
does this work? you might need to do `if np.isinf(mass_balance)`. please verify

Reply:
yep this works!